File: use-cases/school-timetabling/src/main/java/org/acme/schooltimetabling/persistence/LessonRepository.java
package org.acme.schooltimetabling.persistence;

import jakarta.enterprise.context.ApplicationScoped;

import org.acme.schooltimetabling.domain.Lesson;

import io.quarkus.hibernate.orm.panache.PanacheRepository;

@ApplicationScoped
public class LessonRepository implements PanacheRepository<Lesson> {

}

File: technology/kotlin-quarkus/src/main/kotlin/org/acme/kotlin/schooltimetabling/persistence/LessonRepository.kt
package org.acme.kotlin.schooltimetabling.persistence

import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepository
import org.acme.kotlin.schooltimetabling.domain.Lesson
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class LessonRepository : PanacheRepository<Lesson>
